kge/util/metric.py

```diff
@@ -0,0 +1,36 @@
+from kge import Config
+from kge.job import Job
+from typing import Union, Iterable, List
+
+
+class Metric:
+    "Utility class for comparing metrics."
+
+    def __init__(self, metric_max: Union[Job, Config, bool]):
+        "Params: metric_max=True means higher is better."
+        if isinstance(metric_max, Job):
+            metric_max = metric_max.config
```

Comment:
Isn't the .get("valid.metric_max") missing here?

Reply:
The line right below does this.